docs/content/release_notes/csi_rn_supported_orchestration.md

```diff
@@ -4,9 +4,10 @@ The following table details orchestration platforms suitable for deployment of t
 
 |Orchestration platform|Version|Architecture|
 |----------------------|-------|------------|
-|Kubernetes|1.21|x86|
 |Kubernetes|1.22|x86|
+|Kubernetes|1.23|x86|
 |Red Hat OpenShift|4.8|x86, IBM Z, IBM Power Systems<sup>1</sup>|
+|Red Hat OpenShift|4.9|x86, IBM Z, IBM Power Systems<sup>1</sup>|
```

Comment:
I think 1.8.0 will also support ocp 4.7

Reply:
@roysahar  ? Please confirm which OCP versions will be supported in 1.8.0. Thank you

Reply:
@rivkap yes also OCP 4.7.

Reply:
@oriyarde  updated in https://github.com/IBM/ibm-block-csi-driver/pull/416. Note that I only updated the supported orchestration and will update the matrix page accordingly. Not the what's new sections - these were previously supported in 1.6.0. unless there is a specific reason / customer why we need to point it out to such an extent.

Reply:
we have "additional support for k8s 1.23" in 1.8.0 changelog, so I assume it's not only about what is completely new, but changes for those who update from previous version